### DISCUSSION TITLE
中國量子計算超越Google

### DISCUSSION THREAD

**Discussion by 109802503gloria** [1101-GS4719-C/Python-Community-GS4719-C#135] [General]
中國科學院所研發的「祖沖之二號」，在超導量子和光量子兩種系統的計算達到新的里程碑，對「量子隨機取樣」任務比目前最快的超級電腦快一千萬倍，計算複雜度比Google的超導量子計算原型機「懸鈴木」高一百萬倍，未來期望實現量子糾錯，用於解決重大應用價值的科學問題。

[比超級電腦快千萬倍中國量子計算超越 google 同類達新里程碑](https://udn.com/news/story/7086/5844115)

**Comment by enoch-lightning** [comment]
中國在人工智慧的領域真的是非常出眾。像是人臉監控讓我們在中國無所遁形。就算有些人可能不喜歡中國，但其實力確實不容小覷。我們也要加油才是，在良性競爭中彼此成長。

**Comment by Jie2222** [comment]
不太懂**量子計算優越性**是甚麼意思，感覺好抽象，無法理解

**Comment by Isongzhe** [comment]
不太了解量子計算的意義，不過感覺很厲害

**Comment by cody910120** [comment]
中國果真是世界強國，但我還是愛逮玩

**Comment by YuHengChen54** [comment]
雖然其他地方跟Google還差地遠，但這量子計算超越Google真的蠻強的

**Comment by JAIMEHUNG** [comment]
感覺會是以後的趨勢，很期待它未來的運用。

**Comment by 109302013evantsao** [comment]
對於是否真的超越google我保持保留的態度，畢竟中國在這方面的事，對外宣稱超越某某某，但實際上只是有理論，並沒有實際時做出來的案例不在少數

**Comment by vvvioletta** [comment]
比較好奇這個量子計算可以應用在什麼領域上面

**Comment by BryanNee** [comment]
對這個領域不是好「了解 感謝同學分享
